src/components/Button/style.scss

```diff
@@ -129,4 +133,7 @@
     transform: translate(-50%, -50%);
     left: 50%;
   }
+  .btn__content {
+    margin: auto;
```

Comment:
if you will use `span` please remove `margin: auto` and add `display: inline-block;`

Reply:
`.btn` has `text-align: center;` so it will be centered